### DISCUSSION TITLE
我在手机上用termux安装了一个Linux，成功的挂起了paimon.bot，每天早上掉线

### DISCUSSION THREAD

**Discussion by Xiaojack1026** [CMHopeSunshine/LittlePaimon#446] [General]
大佬好，我在手机上用termux安装了一个Linux，成功的挂起了paimon.bot，每天早上掉线
![a45f007f177fea59711f1b6d991a3bc](https://github.com/CMHopeSunshine/LittlePaimon/assets/125620242/526b356b-b792-41fe-915d-3be9d377b074)
![732efa802135d609c1aa97c49c9391f](https://github.com/CMHopeSunshine/LittlePaimon/assets/125620242/a37effa6-d3a6-4f92-b4e9-f80f79eee15d)
![35c98384c65e04dd990448185795051](https://github.com/CMHopeSunshine/LittlePaimon/assets/125620242/398a39c1-d910-4e42-afef-7b90444044c3)
就是内置的go-cqhttp，用手机游览器登录，到了晚上2点多就掉线，termux一直是在运行，就是不知道是哪里没设好

**Comment by CMHopeSunshine** [comment]
无更多有用信息，无法判断问题，可以尝试使用gocq本体

**Comment by Xiaojack1026** [comment]
可我用的是安卓手机，本体没有安卓版本

**Reply by CMHopeSunshine** [reply]
怎么会没有呢，插件版也只是下一个本体来帮你运行

**Reply by Xiaojack1026** [reply]
就是插件运行一段时候就掉线了

**Reply by CMHopeSunshine** [reply]
账号掉线那可能是和tx最近的风控策略有关

**Reply by Xiaojack1026** [reply]
对喔，现在动不动就风控
